apps/admin-portal/src/pages/users.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Button, Grid } from "semantic-ui-react";
+import { UserSearch, UsersList } from "../components/users";
+import { InnerPageLayout } from "../layouts";
+
+/**
+ * Users info page.
+ *
+ * @return {JSX.Element}
+ */
+export const UsersPage = (): JSX.Element => {
+    const { t } = useTranslation();
```

Comment:
As a standard, could we change the `t` into a more meaningful name?

Reply:
We are using the i18next library for the internationalization of the application. So using { t } is a commonly used practice when using that library https://github.com/i18next/react-i18next. So I think it would be better if we can adhere to their standard.

Reply:
Standard is `t` as per their documentation. Will keep it as it is.